app/models/curator/digital_object.rb

```diff
@@ -15,6 +15,9 @@ class DigitalObject < ApplicationRecord
 
     scope :for_serialization, -> { includes(:file_sets, exemplary_image_mapping: :exemplary_file_set).with_metastreams }
     scope :for_reindex_all, -> { for_serialization.joins(:administrative, :descriptive, :workflow) }
+    scope :issue_object, -> { where.not(contained_by_id: nil) }
+    scope :with_admin_set_ark, ->(admin_set_ark_id) { joins(:admin_set).where(admin_set: { ark_id: admin_set_ark_id }) }
```

Comment:
Which issue is this line intended to address? Or is it just for convenience?

Reply:
Just convenience. Mostly for querying newspaper objects stuck in derivatives